tools/cpp/cc_configure.bzl

```diff
@@ -79,6 +79,7 @@ cc_autoconf = repository_rule(
         "GCOV",
         "HOMEBREW_RUBY_PATH",
         "SYSTEMROOT",
+        "TERM", # Use to detect if Bazel is running in MSYS.
```

Comment:
Please considering use `MSYSTEM` instead of `TERM`

Reply:
Thanks for the tips! TIL `msys2_shell.bat`, `mingw64_shell.bat` and `mingw32_shell.bat` are not entirely the same.

Reply:
Yes, IIRC they have different PATH value to make different gcc version available by default.